src/app/components/Containers/TextContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Headline from '../Headline';
+import SubHeading from '../SubHeading';
+import {
+  commonTextPropTypes,
+  baseTextDefaultPropTypes,
+} from '../../models/proptypes';
+
+const Blocks = {
+  headline: Headline,
+  subheading: SubHeading,
+};
+
+const TextContainer = blocks => {
+  const { type } = blocks;
+  const text = blocks.model.blocks[0].model.blocks[0].model;
```

Comment:
This variable is slightly misleading, since it contains the model object not just the text string. 
If you destructure this, with `const { text }` on the left hand side, then `text` will contain just the text string. Then in Headline and SubHeading you just need to pass in a single argument, `text`.

Reply:
By doing this, you can also remove the `containerText` function in `/models/blocks` that is needed for tests and stories, since you can just pass in the text string instead.